test/common/index.js

```diff
@@ -804,3 +804,9 @@ exports.runWithInvalidFD = function(func) {
 
   exports.printSkipMessage('Could not generate an invalid fd');
 };
+
+Object.defineProperty(exports, 'isFSSensitiveCI', {
+  get: function() {
+    return process.env.TRAVIS;
```

Comment:
Since this is only used in one test, I anticipate there will either be push back on landing this, or a follow up PR that just inlines it in that single test.

Reply:
I'm looking throw Travis fails, and I'm already up to 3 flaky tests.
Also IMHO this is a classic place for an abstraction, Travis is just one example.

Reply:
I suggest to just create a new file instead of adding it to the main common file.

Reply:
Good idea 👍

Reply:
Re-stating that the above is non-blocking because wow I really want this to land so that Travis will stop blowing up on test-fs-watch. The current way is fine and is easy enough to refactor out the way I'm describing if there ever ends up being consensus to do it that way instead.

Reply:
>  Anyone seeing `if (process.env.TRAVIS)` in a test will likely know exactly what is going on (especially if there's a comment). Seeing `common.hasLimitedFSWatchers()`? Not so much.

Pardon me while I briefly get into a debate with myself.

But, Rich, someone seeing `process.env.TRAVIS` will know that it's checking for Travis but not *why*. `common.hasLimitedFSWatchers()` explains right in the name the reason that we are skipping the test. (And if you don't think so, suggest a better name, Rich!)

Debate over. Thanks. (I'd still prefer inlining, but whatevs.)